test/gulpfile-js/generate.js

```diff
@@ -16,29 +14,28 @@ const debug = require('gulp-debug');
 const gulpIf = require('gulp-if');
 
 
-gulp.task('generate-lots-of-files', function () {
-  return rimraf('test/generated/es7 test/generated/ts build/generated').then(function () {
-    return exec('mkdir -p test/generated/es7');
-  }).then(function () {
-    return exec('mkdir -p test/generated/ts');
-  }).then(function () {
-    return B.all([
-      ..._.times(24).map(function (i) {
-        return exec(`cp test/fixtures/es7/lib/a.es7.js test/generated/es7/a${i + 1}.es7.js`);
-      }),
-      ..._.times(24).map(function (i) {
-        return exec(`cp test/fixtures/ts/lib/b.ts test/generated/ts/b${i + 1}.ts`);
-      }),
-    ]);
-  });
+const spawnWatch = spawnWatcher.use(gulp);
+
+gulp.task('generate-lots-of-files', async function () {
+  await rimraf('test/generated/es7 test/generated/ts build/generated');
+  await exec('mkdir -p test/generated/es7');
+  await exec('mkdir -p test/generated/ts');
+  await B.all([
+    ..._.times(24).map(function (i) {
```

Comment:
I'd use parentheses here to ease code reading

Reply:
I don't know where such parentheses would go to make this more readable.

Reply:
```
...(
  _.times(24).map
)
```